src/Illuminate/Auth/Notifications/ResetPassword.php

```diff
@@ -114,7 +114,7 @@ public static function createUrlUsing($callback)
     /**
      * Set a callback that should be used when building the notification mail message.
      *
-     * @param  \Closure  $callback
+     * @param  \Closure(mixed, string): \Illuminate\Notifications\Messages\MailMessage  $callback
```

Comment:
It should also be possible to return a Mailable (I have never tried that).
```suggestion
     * @param  \Closure(mixed, string): (\Illuminate\Notifications\Messages\MailMessage|\Illuminate\Mail\Mailable)  $callback
```